### ISSUE TITLE
No one noticed that once the screen was turned off or the phone was restarted Won't it be reconnected?

### ISSUE THREAD

**Issue by goodman-xh** [qH0sT/AndroSpy#73]
First of all, I'm sure the author's contribution. This is a great rat. If it can solve the problem of disconnection caused by lock screen and restart, it's too damn perfect

**Comment by qH0sT** [comment]
> First of all, I'm sure the author's contribution. This is a great rat. If it can solve the problem of disconnection caused by lock screen and restart, it's too damn perfect

what is the phone model? chinese brands like huawei, oppo, xiomi etc.? because This problem only available on this phones that have chinese brand. (I have tested)

**Comment by qH0sT** [comment]
And be sure settings is correct in Battery Settings of Phone (Allow Background Working, Allow Autostart etc.)

**Comment by nurialibal** [comment]
ghost tht yazdım sana ulaşmak için ama yanlış birine yazmışım yardımın gerek üstad megaistanbul1453@gmail.com bana ulaşırsan sevinirim babacan

**Comment by AngelBlaizeC** [comment]
Yeah i had this issue before mine is (OPPO). so i did research and i have to uninstall some background killer via ADB in order for the rat not to close while running in background.. as of now its running perfectly.. but there are times it no longer connect after 5-6 days... i have to reinstall androspy for it to work again thou.

**Comment by qH0sT** [comment]
> Yeah i had this issue before mine is (OPPO). so i did research and i have to uninstall some background killer via ADB in order for the rat not to close while running in background.. as of now its running perfectly.. but there are times it no longer connect after 5-6 days... i have to reinstall androspy for it to work again thou.

I have fixed this issue, this problem was also available for me. I have fixed AlarmManager, now it is working infinity without any problem on phone. Use latest edition update and check for same problem.

**Comment by goodman-xh** [comment]
Yes, Chinese brands
vivo x9
I use the latest version, as long as the lock screen or restart will disconnect
Your work is great. You're a great person

**Comment by goodman-xh** [comment]
And I found a problem
I generated APK on windows, but I didn't distribute it to anyone
Incredible is that occasionally a few minutes there will be a mobile phone automatically connected, I can actually operate this mobile phone
。。。。 I have a guess. It's not the author, is it

**Comment by Kevin2021-jk** [comment]
The problem of restarting and not going online actually exists in every version, and no specific reason has been found. I can only wait for the author to update it!

**Comment by qH0sT** [comment]
> The problem of restarting and not going online actually exists in every version, and no specific reason has been found. I can only wait for the author to update it!

In the phones and tablet that I tested for boot, trojan working on boot without any problem. Did you allow AutoStart in Settings of battery? I have been using it for a long time without any problems.

And can you edit this file (BootReceiver.cs) with this codes in link: https://paste.ubuntu.com/p/ZFRHnfD6HM/
and build a new client with new BootReceiver.cs
I added also new IntentFilter: "android.intent.action.QUICKBOOT_POWERON" and removed "if block" from OnReceive method.

check for same problem and please let me know.

**Comment by Kevin2021-jk** [comment]
After the master exits and then turns on, the client will not go online.

**Comment by Kevin2021-jk** [comment]
The screen view pops up, what's the problem?
![20210324144219](https://user-images.githubusercontent.com/62233800/112268324-c066e180-8cb1-11eb-9e67-f09abc703a08.jpg)

**Comment by qH0sT** [comment]
> The screen view pops up, what's the problem?
> ![20210324144219](https://user-images.githubusercontent.com/62233800/112268324-c066e180-8cb1-11eb-9e67-f09abc703a08.jpg)

Live Screen works with MediaProjection API of Android. And MediaProjection API has been available since API Level 21; you can't use Live Screen under API Level 21; The API Level of Victim must be >= 21.
